### DISCUSSION TITLE
Enable custom registries for datasource "golang-version"

### DISCUSSION THREAD

**Discussion by mikaello** [renovatebot/renovate#17787] [Request Help] [answered]
Custom registries are disabled for datasource `golang-version`:

https://github.com/renovatebot/renovate/blob/66f88bed7c7c7b637e5670fdc07ffbe09304b112/lib/modules/datasource/golang-version/index.ts#L30

Is there a reason for this, or could this be switched to `true`?

We are using Artifactory for all our communications with internet, so I have added a working remote, `https://artifactory.some-name.com/github-usercontent/golang/website/`, but this is refused by our on-premises hosted Renovate because it is disabled by the mentioned config:

```json
{
   "msg": "Custom registries are not allowed for this datasource and will be ignored",
   "name": "renovate",
   "registryUrls": [  "https://artifactory.some-name.com/github-usercontent/golang/website/" ],
   "repository": "my/repo"
}
```

**Comment by rarkins** [comment, marked as answer]
PR welcome to switch this

**Reply by mikaello** [reply]
PR: https://github.com/renovatebot/renovate/pull/17828
